### PR TITLE
Update to latest AutoForm and remove a old fix

### PR DESCRIPTION
**Pull request by bySabi**
AutoForm version 5.7.1 fix the old form schema reactivity bug.
Please review @Pagebakers 
This have expose WizardRouter too. Feel free to remove if you think is not OK

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -29,5 +29,8 @@ Package.onUse(function(api) {
   ], 'client');
 
   if (api.export)
-    api.export('Wizard', 'client');
+    api.export([
+      'Wizard',
+      'WizardRouter'
```
Comment (Pagebakers) [inline package.js]: This is part of a separate PR, can you remove this?

File: package.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ Package.onUse(function(api) {
     'localstorage'
   ], 'client');
 
-  api.use('aldeed:autoform@5.0.0', 'client');
+  api.use('aldeed:autoform@5.7.1', 'client');
```
Comment (Pagebakers) [inline package.js]: use @5.7.0, so minor versions get automatically updated.

File: package.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: 'forwarder:autoform-wizard',
   summary: 'A multi step form component for AutoForm.',
-  version: '0.8.0',
+  version: '0.8.2',
```
Comment (Pagebakers) [inline package.js]: Let's make this 0.9.0 since this will incompatible with older autoform versions.

File: package.js (hunk 2 above)
Comment (bySabi) [inline package.js]: I don´t know that. Thanks for info.

File: package.js (hunk 1 above)
Comment (bySabi) [inline package.js]: Not problem. This change was there before AutoForm update commit. Going to fix now!

File: package.js (hunk 3 above)
Comment (bySabi) [inline package.js]: No problem. I prefer that you put numbers. Sorry!

**Comment by bySabi** [comment]
Now is done!.

**Comment by Pagebakers** [comment]
Thanks a bunch!

**Comment by bySabi** [comment]
This PR don´t touch package number, still on `0.8.0`

On README is the same typo that in flowrouter package:

```
  path: function(name, stepId) {
    return FlowRouter.go(name, this.getParams(stepId));
  }
```

must be

```
  path: function(name, stepId) {
    return FlowRouter.path(name, this.getParams(stepId));
  } 
```

Or maybe remove all FlowRouter code implementantion example from README . I thinks is not needed ther anymore. But is your decission.

**Comment by Pagebakers** [comment]
I'll leave it in there for now, because it's still not sure which router will be the officially supported router.

**Comment by bySabi** [comment]
@Pagebakers please update package version

**Comment by Pagebakers** [comment]
Published!

**Pull request merged**